lib/datasource/github-releases/index.ts

```diff
@@ -13,9 +15,20 @@ const cacheNamespace = 'datasource-github-releases';
 
 const http = new GithubHttp();
 
-function getCacheKey(depHost: string, repo: string): string {
+function getReleasesCacheKey(registryUrl: string, repo: string): string {
   const type = 'tags';
-  return `${depHost}:${repo}:${type}`;
+  return `${registryUrl}:${repo}:${type}`;
+}
+
+function getSourceUrlBase(registryUrl: string): string {
+  // default to GitHub.com if no GHE host is specified.
+  return ensureTrailingSlash(registryUrl ?? 'https://github.com/');
+}
+
+function getApiBaseUrl(sourceUrlBase: string): string {
+  return sourceUrlBase === 'https://github.com/'
+    ? `https://api.github.com/`
+    : `${sourceUrlBase}api/v3/`;
 }
```

Comment:
Please move simple helper functions to new file `./common.ts`, so this files gets easier to read and maintain

Reply:
Sounds good to me - is this specific to `getSourceUrlBase` and `getApiBaseUrl`, or all non-exported helpers? (e.g. `findDigestFile`)

Reply:
I opted to move these to `./common.ts` and introduce `./digest.ts` for digest-specific helpers. I'm happy to make any further adjustments you suggest!